src/components/atoms/Dropdown.tsx

```diff
@@ -1,29 +1,29 @@
 import React, { useState } from "react";
 
 type Props = {
-  cityNames: string[];
+  values: string[];
```

Comment:
I think instead of having `string[]` as a type, we introduce a new type called `DropdownOption` that contains both a value and label key. Most of the time when we're gonna have a dropdown we're gonna have IDs of items as the values and some text that will show up to the user.